checker/jdk/nullness/src/javax/xml/parsers/FactoryFinder.java

```diff
@@ -285,7 +285,6 @@ private static void dPrint(String msg) {
      *
      * @return instance of provider class if found or null
      */
-    @SuppressWarnings("nullness")    // Checker analysis for return type found : @FBCBottom @Nullable  null but it also returns iterator.next() which can be non-null also. In x.getMessage() parameter found : @Initialized @Nullable String but this will not dereference a null here as underlying exception 'e' is not null. 	
```

Comment:
This removal causes the following error. Therefore, I am adding a suppress warning here.
found   : @FBCBottom @Nullable  null
required: T[ extends @Initialized @Nullable Object super @Initialized @NonNull Void]